src/App.js

```diff
@@ -1,74 +1,21 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
 import AppHeader from "./components/AppHeader";
 import AppContent from "./components/AppContent";
 import ReactDOM from "react-dom";
 import modalConfig from "./utils/modal.config.json";
-import { useUID } from "react-uid";
 
 import "./App.sass";
 import ModalWindow from "./components/UI/ModalWindow/ModalWindow";
+import CardContext from "./components/CardContext";
```

Comment:
Context components/storages are usually set into a separate folder in the project src structures.